### ISSUE TITLE
iocpreactor does not cancel pending operations on shutdown

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10695]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| PenguinOfDoom reported|
|-|-|
|Trac ID|trac#1039|
|Type|defect|
|Created|2005-05-14 11:10:34Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1039 1039
type__defect defect
reporter__PenguinOfDoom PenguinOfDoom
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1116069034000000 1116069034000000
changetime__1244292407000000 1244292407000000
version__ 
owner__ 
cc__exarkun cc__justinj cc__PenguinOfDoom
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|PenguinOfDoom commented|
|-|-|

```
#!html
<pre>
t.i.iocpreactor.proactor.Proactor.removeAll should probably do something useful
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|PenguinOfDoom set status to `closed`|
|-|-|

Implemented in [#1760](https://github.com/twisted/twisted/issues/1760)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
